tools/ecc-certs/makecerts.sh

```diff
@@ -75,11 +102,18 @@ function device {
     -out "${CPID}/${name}-crt.pem" \
     -days ${X509_VALIDITY}
 
+  openssl ${key_conversion_params} -outform der -in "${CPID}/${name}-key.pem" -out "${CPID}/${name}-key.der"
+  openssl x509 -outform der -in "${CPID}/${name}-crt.pem" -out "${CPID}/${name}-crt.der"
+
   f="${CPID}/${name}-c-snippets.txt"
   echo > $f
   pem2c "${CPID}/${name}-key.pem" >> $f
```

Comment:
pem2c function itself and these two steps seem to be rdundant if we're gonna use xxd

Reply:
Thanks for reviewing. 

The 2 are in different formats, so I thought pem2c might be used for other reasons. pem2c outputs the base64 encoded PEM format as a multiline string, whereas xxd is outputting the binary DER format as a C array.  

pem2c =>
```
$ cat blah-c-snippets.txt

"-----BEGIN RSA PRIVATE KEY-----\r\n"
"MIICXwIBAAKBgQDYD+evJfdB/ROPJBE9ZrfpkpvJPt895zGyt7Pz4A7eFUYv/fQL\r\n"
"RBhMq+b5Ht+68ZcmOdXJrg16PZiQmMIN+tzW+QNX1PTr3uKwN6df9jzR9YywqHbr\r\n"
"6mrHzY+8dvrVzYaBBO64PyjmLXmGNCs2YF4zNoc6LOprLJlBOxvMrDpzWwIDAQAB\r\n"
"AoGBALY0O3pNtwT0CeGFVfCExgV4OF3u4k4OwK9pEZ1Qztjj8euY6ovxodhOgxqE\r\n"
============ 8 >< ========
"K0uXzsMkOg5eg8MAtmx1yQJBAJxahq2LhjrxkTTZHVc0P+z6S4TJJvzcdM+bKmzH\r\n"
"aGD04jKe/uCjKSBmU45ziS1AZgtYji6PAug1EOt0c7UoU4s=\r\n"
"-----END RSA PRIVATE KEY-----\r\n"
...
```

xxd =>
```
$ cat blah-c-array.txt

unsigned char avtds_RSA1024_crt_der[] = {
  0x30, 0x82, 0x02, 0x2e, 0x30, 0x82, 0x01, 0xd4, 0x02, 0x01, 0x01, 0x30,
  0x0a, 0x06, 0x08, 0x2a, 0x86, 0x48, 0xce, 0x3d, 0x04, 0x03, 0x02, 0x30,
  0x81, 0x85, 0x31, 0x0b, 0x30, 0x09, 0x06, 0x03, 0x55, 0x04, 0x06, 0x13,
============ 8 >< ========
  0x82, 0x85, 0x99, 0x76, 0x59, 0xa3, 0x04, 0xcf, 0xb5, 0xa1
};
unsigned int avtds_RSA1024_key_der_len = 610;
...
```